django/db/models/query.py

```diff
@@ -935,7 +956,7 @@ def complex_filter(self, filter_obj):
             clone.query.add_q(filter_obj)
             return clone
         else:
-            return self._filter_or_exclude(None, **filter_obj)
+            return self._filter_or_exclude(False, **filter_obj)
```

Comment:
This is unrelated and unnecessary, but seems too small to create a separate PR. As far as I understood, None here is a remnant of some outdated _filter_or_exclude signature.

Reply:
It should at least be made in a different commit.